scipy/optimize/tests/test_linprog.py

```diff
@@ -1955,6 +1991,15 @@ class TestLinprogHiGHSSimplexDual(LinprogHiGHSTests):
     method = "highs-ds"
     options = {}
 
+    def test_lad_regression(self):
+        '''The scaled model should be optimal but unscaled model infeasible.'''
+        c, A_ub, b_ub, bnds = l1_regression_prob()
+        res = linprog(c, A_ub=A_ub, b_ub=b_ub, bounds=bnds,
+                      method=self.method, options=self.options)
+        assert_equal(res.status, 4)
```

Comment:
Please also look for the expected error message, as there are other messages that correspond with status 4.

I'll merge after this and the check for `x` in `bnds` passes CI.